installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc

```diff
@@ -3,6 +3,7 @@
 :context: ipi-install-configuration-files
```

Comment:
Looks like installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc  isn't showing in the preview build. I think  you just need to update the _topic_map.yml file with that assembly.

Reply:
@johnwilkins That's pointing to /installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc and not installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc so I'm a little confused. We can discuss when we meet momentarily, but this should be about ready to go!